service/src/main/kotlin/fi/espoo/evaka/invoicing/domain/FeeDecisions.kt

```diff
@@ -183,11 +183,13 @@ data class FeeDecisionDetailed(
         return isRetroactive(this.validDuring.start, LocalDate.from(sentAtLocalDate ?: LocalDate.now()))
     }
 
+    // TODO: remove
```

Comment:
Either add more descriptive comment when to remove this or remote this and the next todo below

Reply:
My idea was to remove the fields immediately after this PR is in prod. I can make a draft PR to make sure that I won't forget to do it.

Reply:
https://github.com/espoon-voltti/evaka/pull/1038